src/buildHTML.js

```diff
@@ -1020,13 +1024,10 @@ groupTypes.katex = function(group, options) {
         ["mord", "katex-logo"], [k, a, t, e, x], options);
 };
 
-const makeLineSpan = function(className, options) {
-    const baseOptions = options.havingBaseStyle();
-    const line = makeSpan(
-        [className].concat(baseOptions.sizingClasses(options)),
```

Comment:
Are the sizing classes not necessary b/c we're manually figuring setting the `borderBottomWidth`?